ubi9-py39/Dockerfile

```
@@ -0,0 +1,49 @@
+# Thoth's extension to OpenShift's S2I build
+FROM registry.access.redhat.com/ubi9/python-39@sha256:6329d5779e89630928d5fa65a7b7b824f6bf41357f5db1688f0efafcbe48b14d
+
+ENV SUMMARY="Thoth's Source-to-Image for Python ${PYTHON_VERSION} applications" \
+    DESCRIPTION="Thoth's Source-to-Image for Python ${PYTHON_VERSION} applications. This toolchain is based on Red Hat UBI9. It includes pipenv." \
+    THOTH_S2I_NAME=quay.io/thoth-station/s2i-thoth-ubi9-py39 \
+    THOTH_S2I_VERSION=0.34.1 \
+    THAMOS_NO_PROGRESSBAR=1 \
+    THAMOS_NO_EMOJI=1 \
+    MICROPIPENV_NO_LOCKFILE_PRINT=0 \
+    MICROPIPENV_NO_LOCKFILE_WRITE=0
+
+LABEL summary="$SUMMARY" \
+    description="$DESCRIPTION" \
+    io.k8s.description="$DESCRIPTION" \
+    io.k8s.display-name="Thoth Python 3.9-ubi9 S2I" \
+    io.openshift.expose-services="8080:http" \
+    io.openshift.tags="python,python39" \
+    name="$THOTH_S2I_NAME:v$THOTH_S2I_VERSION" \
+    vendor="AICoE at the Office of the CTO, Red Hat Inc." \
+    authoritative-source-url="https://quay.io/thoth-station/s2i-thoth" \
+    version="$THOTH_S2I_VERSION" \
+    release="0" \
+    ninja.thoth-station.version="0.7.0-dev" \
+    maintainer="Thoth Station <aicoe-thoth@redhat.com>"
+
+USER 0
+COPY ./s2i_assemble.patch /tmp/s2i_assemble.patch
+COPY ./requirements.txt $HOME/requirements.txt
+RUN TMPFILE=$(mktemp) && \
+    TMPFILE_ASSEMBLE=$(mktemp) && \
+    pushd "${STI_SCRIPTS_PATH}" && \
+    ls /tmp/s2i_assemble.patch && \
+    patch -p 1 </tmp/s2i_assemble.patch && popd && \
```

```suggestion
    pushd "${STI_SCRIPTS_PATH}" && patch -p 1 </tmp/s2i_assemble.patch && popd && \
```

lets remove the ls from this.